tests/all_tests_v2.nim

```diff
@@ -19,6 +19,10 @@ import
   ./v2/waku_archive/test_retention_policy,
   ./v2/waku_archive/test_waku_archive
 
+# TODO: add the postgres test when we can mock the database.
+# The postgres tests now need a running postgresql database running locally.
+#  ./v2/waku_archive/test_driver_postgres,
```

Comment:
If you wanted the tests enabled, you could just execute a `docker run` command to start postgres container and then clean it up.

GH Actions support docker, so it should work and most of people (us) running tests also have docker installed and working.